app/components/LeftPaneEntries/index.tsx

```diff
@@ -396,23 +398,6 @@ function LeftPane(props: Props) {
         }
     }, [leadId, onEntryCreate]);
 
-    const handleAttachmentClick = useCallback((attachment: LeadPreviewAttachmentType) => {
-        if (onEntryCreate) {
-            onEntryCreate({
-                clientId: randomString(),
-                entryType: 'ATTACHMENT',
-                lead: leadId,
-                leadAttachment: attachment.id,
-                excerpt: '',
-                droppedExcerpt: '',
-            });
-            setLeadAttachmentsMap((oldValue) => ({
-                ...oldValue,
-                [attachment.id]: attachment,
-            }));
-        }
-    }, [leadId, onEntryCreate]);
-
```

Comment:
Is there a reason we're handling this in 'EntryEdit' and not here?

Reply:
@subinasr  By adding handleAttachmentClick to the LeftPane Entries, the leadId is utilized in both the secondary and review tabs. Moving it to entryEdit makes the leadId accessible across all tabs, which explains why the image was only visible in the primary tab previously.